test/postgresql.test.js

```diff
@@ -263,6 +263,18 @@ describe('postgresql connector', function() {
       });
   });
 
+  it('should support where filter for array type field', function(done) {
```

Comment:
We prefer to use async/await style for all newly written tests. Can you please rewrite your test as follows?

```js
it('should support where filter for array type field', async () => {
  const post = await Post.find(/* your filter */);
  should.exist(post);
});
```

Reply:
Done with this test-case change